src/sentry/api/endpoints/project_users.py

```diff
@@ -42,3 +43,30 @@ def get(self, request, project):
             paginator_cls=DateTimePaginator,
             on_results=lambda x: serialize(x, request.user),
         )
+
+    def delete(self, request, project):
+        """
+        Delete an Event User
+        ````````````````````````````````
+
+        Delete an event's user.
+
+        :pparam string organization_slug: the slug of the organization.
+        :pparam string project_slug: the slug of the project.
+        :qparam string query: Prefix ``id`` should be used to match on.
+                              For example, ``query=id:3``
```

Comment:
Will this always just use `id`:? if so, maybe we just just pass it directly as `ident` rather than parsing the `id:3` here.

Reply:
Actually wondering if we should just pass the actual row id. Alternatively, maybe this should be in src/sentry/api/endpoints/project_user_details.py and just use the hash, so that it matches how we fetch users from that endpoint.

Our usual pattern for endpoints is that operations on a specific item (get/edit/delete) go in `_details` and operations that operate on the list (fetch items, create) go in `_index`, or in this case, just the unprefixed file,

Reply:
Ahh got it, moved this to `project_user_details.py` to use the hash